oak/server/rust/oak_runtime/src/lib.rs

```diff
@@ -108,6 +112,14 @@ pub struct GrpcConfiguration {
     pub grpc_client_root_tls_certificate: Option<Certificate>,
 }
 
+/// Configuration options related to Wasm module signatures.
+/// Consists of parsed `minisign::PublicKey` and `minisign::SignatureBox`.
```

Comment:
Probably worth a comment saying that the signatures in this struct are not guaranteed to have been verified already? Otherwise if anyone assumes that they are verified, it will break the entire trust assumptions.